File: Answers/07_while/e05_product.cs
using System;

class Program
{
    static void Main()
    {
        int start = int.Parse(Console.ReadLine());
        int product = 1;
        for (int i = start; i < start + 7; i++)
        {
            product *= i;
        }
        Console.WriteLine(product);

        start = int.Parse(Console.ReadLine());
        product = 1;
        int j = start;
        while (j < start + 7)
        {
            product *= j;
            j++;
        }
        Console.WriteLine(product);
    }
}

File: Answers/07_while/e05_product.py
start = int(input("Zadajte počiatočné číslo: "))
product = 1
for i in range(start, start + 7):
    product *= i
print(product)


start = int(input("Zadajte počiatočné číslo: "))
product = 1
i = start
while i < start + 7:
    product *= i
    i += 1
print(product)
